task/02-numbers-tasks.js

```diff
@@ -221,20 +229,24 @@ function isPrime(n) {
  *   toNumber(new Number(42), 0) => 42
  */
 function toNumber(value, def) {
-    throw new Error('Not implemented');
+	const valueToNumber = Number(value);
+	if (valueToNumber) {
+		return valueToNumber;
+	}
+	return def;
 }
 
 module.exports = {
-    getRectangleArea: getRectangleArea,
-    getCicleCircumference: getCicleCircumference,
-    getAverage: getAverage,
-    getDistanceBetweenPoints: getDistanceBetweenPoints,
-    getLinearEquationRoot: getLinearEquationRoot,
-    getAngleBetweenVectors: getAngleBetweenVectors,
-    getLastDigit: getLastDigit,
-    parseNumberFromString: parseNumberFromString,
-    getParallelipidedDiagonal: getParallelipidedDiagonal,
-    roundToPowerOfTen: roundToPowerOfTen,
-    isPrime: isPrime,
-    toNumber: toNumber
+	getRectangleArea: getRectangleArea,
+	getCicleCircumference: getCicleCircumference,
+	getAverage: getAverage,
+	getDistanceBetweenPoints: getDistanceBetweenPoints,
+	getLinearEquationRoot: getLinearEquationRoot,
+	getAngleBetweenVectors: getAngleBetweenVectors,
+	getLastDigit: getLastDigit,
+	parseNumberFromString: parseNumberFromString,
+	getParallelipidedDiagonal: getParallelipidedDiagonal,
```

Comment:
the PR has redundant changes in styling, spaces, comments....
Please follow the simple rules in the future:
 - do not add any non logic changes to code if review is required
 - if the changes in styling are important - you can do it as separate PR with styling only changes to avoid noise in the main PR